frontends/web/src/components/pillbuttongroup/pillbuttongroup.tsx

```diff
@@ -21,6 +21,7 @@ import styles from './pillbuttongroup.module.css';
 type TPillTabProps = {
   children: ReactNode;
   className?: string;
+  size?: 'md' | 'lg';
```

Comment:
small nit: I find this more readable
```suggestion
  size?: 'medium' | 'large';
```